src/accelerate/big_modeling.py

```diff
@@ -463,6 +464,13 @@ def wrapper(*args, **kwargs):
         else:
             model.cuda = add_warning(model.cuda, model)
 
+        # Check if we are using multi-gpus with RTX 4000 series
+        use_multi_gpu = len([device for device in set(device_map.values()) if device not in ("cpu", "disk")]) > 1
+        if use_multi_gpu and not check_cuda_p2p_ib_support():
+            logger.warning(
+                "RTX 4000 series have issues with P2P. This can affect the multi-gpu inference when using accelerate device_map."
+                "Please make sure to update your driver to the latest version"
```

Comment:
Given what https://github.com/huggingface/accelerate/pull/2558 introduces, we can clean this up a bit!
```suggestion
                "We've detected an older driver with an RTX 4000 series GPU. These drivers have issues with P2P. "
                "This can affect the multi-gpu inference when using accelerate device_map."
                "Please make sure to update your driver to the latest version"
```

Reply:
awesome !